src/gluonts/mx/block/cnn.py

```diff
@@ -53,20 +53,22 @@ class CausalConv1D(gluon.HybridBlock):
     def __init__(
         self,
         channels: int,
-        kernel_size: int,
-        dilation: int = 1,
+        kernel_size: Union[int, Tuple[int], List[int]],
```

Comment:
We could give this type a name.

Reply:
not sure, because then anyone looking into the docstring will find some custom type without knowing what it is

unless we document that too, but a `Union` is pretty self-explanatory and not too bad I think